Sample-Scripts-and-Notebooks/Official/Secured Locker/azure-functions/README.md

```diff
@@ -15,19 +15,19 @@ To run the sample, you need:
 * Language : Python
 * Python interpreter: Python 3.6.x, 3.7.x 3.8.x are supported
 * Template: Azure Blob Storage trigger
-* Select a storage account: choose your SCZ-MM service storage account (i.e. sczmm1models)
+* Select a storage account: choose your Azure-Percept-SMM service storage account (i.e. testmmmodels)
 * Blob storage path to be monitored: data
 
 ## 2. Add the following environment variables in local.settings.json with proper settings:
 ```
 "AZURE_CLIENT_ID": "", 
 "AZURE_CLIENT_SECRET": "",
 "AZURE_TENANT_ID": "",
-"scz_mm_server_url": "",
-"scz_mm_storage_account": "",
-"scz_mm_telemtry_storage_container": "data",
-"scz_mm_model_name": "",
-"scz_mm_model_version": "",
+"mm_server_url": "",
+"mm_storage_account": "",
+"mm_telemtry_storage_container": "data",
```

Comment:
telemtry is missing an e, it should be tel**e**metry

Reply:
Thanks. Fixed in the new revision.